packages/capacitor-notifications/src/index.ts

```
@@ -0,0 +1,448 @@
+import type {
+  CapgoBackgroundNotificationEvent,
+  CapgoNotificationEvent,
+  CapgoNotificationInstallMode,
+  CapgoNotificationOpenedEvent,
+  CapgoNotificationPermission,
+  CapgoNotificationPlatform,
+  CapgoNotificationProvider,
+  CapgoNotificationRegisterOptions,
+  CapgoNotificationRegistration,
+  CapgoNotificationToken,
+  CapgoNotificationsConfig,
```

ESLint

[error] Expected "CapgoNotificationsConfig" to come before "CapgoNotificationToken".

(perfectionist/sort-named-imports)